dev-support/Jenkinsfile_GitHub

```diff
@@ -168,6 +168,7 @@ pipeline {
                         BUILD_URL_ARTIFACTS = "artifact/${WORKDIR_REL}/${PATCH_REL}"
                         DOCKERFILE = "${WORKDIR}/${DOCKERFILE_REL}"
                         YETUS_DRIVER = "${WORKDIR}/${YETUS_DRIVER_REL}"
+                        SKIP_ERRORPRONE = true
```

Comment:
default is enable, so skip for "yetus jdk8 Hadoop3 checks"?

Reply:
Yes, will skip for jdk8 and jdk11 checks. Only enable it in general check.